MatrixKit/Models/Account/MXKAccountManager.m

```diff
@@ -17,6 +17,10 @@
 
 #import "MXKAccountManager.h"
 
+static NSString *const kMXKAccountsKey = @"kMXKAccountsKey";
+
+static NSString *const kMXAppGroupID = @"group.org.matrix";
```

Comment:
I aggree on moving it to MXKAppSettings, but about the property.. maybe in some places we are not going to use the shared userdefaults in the app?